stripe/http_client.py

```diff
@@ -144,6 +144,12 @@ def _handle_request_error(self, e):
 class UrlFetchClient(HTTPClient):
     name = 'urlfetch'
 
+    def __init__(self, verify_ssl_certs=True, deadline=55):
```

Comment:
Do you think it might be worth just putting `deadline` in the existing constructor for `HTTPClient`? Avoids duplicating the `verify_ssl_certs` setting logic which is kind of nice.